oem/firstvoices/ios/build_common.sh

```diff
@@ -186,25 +191,37 @@ if [ $CODE_SIGN = true ]; then
   if [ $DO_ARCHIVE = true ]; then
     # Time to prepare the deployment archive data.
     echo ""
-    echo "Preparing .ipa file for deployment."
+    echo "Preparing .ipa file for deployment to real devices."
     xcodebuild $XCODEFLAGS_EXT -scheme $TARGET -archivePath $ARCHIVE_PATH archive -allowProvisioningUpdates \
                VERSION=$VERSION \
                VERSION_WITH_TAG=$VERSION_WITH_TAG
 
     assertDirExists "$ARCHIVE_PATH"
 
+
+    # Do NOT use the _EXT variant here; there's no scheme to ref, which will lead
+    # Xcode to generate a build error.
     xcodebuild $XCODEFLAGS -exportArchive -archivePath $ARCHIVE_PATH -exportOptionsPlist exportAppStore.plist \
                -exportPath $BUILD_PATH/${CONFIG}-iphoneos -allowProvisioningUpdates \
                VERSION=$VERSION \
                VERSION_WITH_TAG=$VERSION_WITH_TAG
   else
-    xcodebuild $XCODEFLAGS -scheme "$TARGET" \
+    xcodebuild $XCODEFLAGS_EXT -scheme "$TARGET" \
                VERSION=$VERSION \
                VERSION_WITH_TAG=$VERSION_WITH_TAG
   fi
 else
   xcodebuild CODE_SIGN_ENTITLEMENTS="" CODE_SIGNING_ALLOWED="NO" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO \
-             $XCODEFLAGS -scheme "$TARGET" \
+             $XCODEFLAGS_EXT -scheme "$TARGET" \
+             VERSION=$VERSION \
+             VERSION_WITH_TAG=$VERSION_WITH_TAG
```

Comment:
It strikes me that we could split all parameters onto their own lines here in the future to make it clearer? But no need to do this today.